lib/cdo/user_helpers.rb

```diff
@@ -62,4 +62,19 @@ def self.sponsor_message(user)
   def self.age_from_birthday(birthday)
     ((Date.today - birthday) / 365).to_i
   end
+
+  AGE_CUTOFFS = [18, 13, 8, 4].freeze
+
+  # Return the highest age range for the given birthday, e.g.
+  # 18+, 13+, 8+ or 4+
+  def self.age_range_from_birthday(birthday)
+    age = age_from_birthday(birthday)
+    age_cutoff = AGE_CUTOFFS.find {|cutoff| cutoff <= age}
+    age_cutoff ? "#{age_cutoff}+" : nil
+  end
```

Comment:
These tests existed indirectly, as tests on the `ProjectsList.student_age_range`. I've inlined `ProjectsList.student_age_range` and moved the tests here.